src/test/java/com/afornalik/ox/view/TestUIConsole.java

```diff
@@ -24,7 +22,7 @@ public void shouldPrintOutGivenString() {
         Scanner scanner = new Scanner(System.in);
```

Comment:
Test który się nazywa "shouldPrintOutGivenString" nie ma w sekcji "given" żadnego napisu. Do tego drukuje "dummy string" zapisany gdzieś poza testem? I to jest jakiś napis bez znaczenia? Hmmm. Po co ten test?

Reply:
Chciałem przetestować metodę print z klasy UISimple, która przysłania Scanner.print();. Ten test jest w zasadzie niepotrzebny, bo testuje jave. Dummy string jest to napis, który miał służyć do porównania. Zmieniłem nazwę.

Reply:
Jeśli test testuje Jawę, to go usuńmy, a nie tylko zmieńmy mu nazwę. :D

Reply:
Racja, był to pierwszy test w tej klasie, powinienem go wcześniej usunąć. Skasowany.